app/src/main/res/layout/activity_session.xml

```diff
@@ -79,10 +83,37 @@
                         android:textSize="24sp"
                         android:id="@+id/session_create_date"/>
                 </android.support.design.widget.TextInputLayout>
+
+                <android.support.design.widget.TextInputLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:paddingTop="16dp">
+                    <android.support.design.widget.TextInputEditText
```

Comment:
Is there a reason you are not using the date/time picker widgets?

Reply:
They were more work to implement and we decided that a textbox could do the job for now.